db/celebrity.js

```diff
@@ -44,16 +31,32 @@ module.exports.init = function(){
             } else {
                 var $ = res.$;
                 var list = $("div.div-col> ul> li>[href *= 'wiki']a").toArray();
-                var stars =[]
+                var celebName = [];
+                var stars = [];
+
+                // Get a list of celebrity names
                 list.map(celeb => {
-                    stars.push(celeb.attribs.title);
+                    celebName.push(celeb.attribs.title);
                 })
-                console.log(stars);
-            }
+
+                // Transfrom names into Celebrity schema and store it into stars 
+                celebName.map(star => {
+                    var celebSchema = {name: `${star}`};
+                    stars.push(celebSchema)
```

Comment:
why are you using two maps while you can do it in a single one without `celebName`